strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PathNupkg.java

```diff
@@ -0,0 +1,182 @@
+package org.carlspring.strongbox.artifact.coordinates;
```

Comment:
I think we don't need it to be serializable for now (and not sure that we will need it in the future). So I think we can ignore this.